src/App.php

```diff
@@ -203,7 +212,8 @@ protected function configureIoc( \Slim\Helper\Set $container ) {
 	protected function configureView( View $view ) {
 		$view->replace( [
 			'app' => $this->slim,
-			'i18nCtx' => $this->slim->i18nContext
+			'i18nCtx' => $this->slim->i18nContext,
+			'supportedLanguages' => $this->getSupportedLanguages(),
```

Comment:
We don't have any coding conventions against trailing commas in PHP, in fact, this is probably the more common style in MediaWiki as it helps to prevent accidental syntax errors when adding new items to arrays. Of course, you have to be careful not to do this in JavaScript though.